### ISSUE TITLE
Update the favicon with our preferred icon

### ISSUE THREAD

**Issue by chipaudette** [open-ultrasound/open-ultrasound.github.io#2]
Our favicon: https://github.com/open-ultrasound/open-ultrasound.github.io/blob/master/favicon.png

Currently, it's the old OpenImage icon.  Update it to the new icon.

**Comment by jedwilbur** [comment]
@chipaudette yes. But seeing how I couldn't get the favicon to show on the webpage....

**Comment by jedwilbur** [comment]
I should probably make one with a transparent background.
